.github/workflows/regression-tests.yml

```
@@ -94,3 +95,24 @@ jobs:
           Pkg.instantiate()
           # Build the docs
           include(joinpath(doc_path, "make.jl"))
+
+  test-julia-manual-build:
+    name: "Julia Manual"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: julia-actions/setup-julia@v2
+        with:
+          version: 'nightly'
+      - uses: actions/cache@v4
+        env:
+          cache-name: cache-artifacts
+        with:
+          path: ~/.julia/artifacts
+          key: ${{ runner.os }}-test-${{ env.cache-name }}-${{ hashFiles('**/Project.toml') }}
+          restore-keys: |
+            ${{ runner.os }}-test-${{ env.cache-name }}-
+            ${{ runner.os }}-test-
+            ${{ runner.os }}-
```

Can't this just use `julia-actions/cache` ?
```suggestion
      - uses: julia-actions/cache@v2
```

---

This makes sense. I think we should update the other places in the regression tests workflow too.

---

It has been updated everywhere else by now